hypothesis-python/src/hypothesis/internal/cache.py

```diff
@@ -61,11 +72,15 @@ def __init__(self, max_size):
         # reordered without updating the index.
         self.keys_to_indices = {}
         self.data = []
+        self.__pinned = 0
```

Comment:
```python
# Total number of pinned entries in this cache.
# (An individual entry pinned multiple times counts as one entry.)
```

Personally I also prefer a more property variable name, like `__pinned_entries` or `__pinned_entry_count`, since it makes the manipulation code a lot more self-describing.